Add simple KPI metric visualization notebook

hemajv opened the pull request:
## Related Issues and Dependencies
#217
#275 

## This introduces a breaking change

- [ ] Yes
- [X] No

<!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->

## Description

Adding a simple KPI metric visualization notebook that plots a time series graph for each metric aggregated over all TestGrid dashboards, jobs and tests i.e. aggregated to the entire platform level

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **durandom** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @durandom` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aicoe-aiops%2Focp-ci-analysis).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["durandom"]} -->

isabelizimm commented on notebooks/data-sources/TestGrid/metrics/metric_visualization_generic.ipynb:
```diff
@@ -0,0 +1,1309 @@
+{
```
<p>small nit: can the graphs with overlapping date labels be rotated for readability?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

hemajv replied:
updated :+1:

aakankshaduggal reviewed (commented):
Overall looks good to me. Just a small suggestion, if you could fix the ticks and add a conclusion to the notebook it will be great.

chauhankaranraj reviewed (commented):
/lgtm :+1:

hemajv commented:
> Overall looks good to me. Just a small suggestion, if you could fix the ticks and add a conclusion/future steps to the notebook it will be great.

thanks for pointing that out! updated the graph ticks and added a conclusion for the notebook as well as the graphs.

MichaelClifford commented:
/approve

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aicoe-aiops/ocp-ci-analysis/pull/290#issuecomment-863176746" title="Approved">MichaelClifford</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aicoe-aiops%2Focp-ci-analysis).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/OWNERS)~~ [MichaelClifford]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

MichaelClifford commented:
:rocket: :rocket: :rocket:  Thanks @hemajv this is great!
